src/locale/languages/nl.js

```diff
@@ -90,6 +90,7 @@ module.exports = {
   'No contributors found': 'Geen medewerkers',
   'No description provided.': 'Niet voorzien van een beschrijving.',
   'No issues': 'Geen issues',
+  'No members found': '',
```

Comment:
Hi @RolfKoenders 

Can you please add a translation of this new phrase?

Reply:
@RolfKoenders I will merge it first. And anybody can feel free to translate those missing words later.